build.assets/makefiles/master/k8s-master/kube-kubelet.service

```diff
@@ -22,6 +22,8 @@ ExecStart=/usr/bin/kubelet \
         --system-reserved=${KUBE_SYSTEM_RESERVED} \
         --cgroup-root=${KUBE_CGROUP_ROOT} \
         --config=/etc/kubernetes/kubelet.yaml \
+        --serialize-image-pulls=false \
```

Comment:
At least this should be set in kubelet configuration file [here](https://github.com/gravitational/planet/blob/kevin/master/1167-image-pull-deadline/tool/planet/constants.go#L471) (as the CLI option is marked as deprecated):
```go
// KubeletConfig specifies the default kubelet configuration
var KubeletConfig = kubeletconfig.KubeletConfiguration{
	// ...
	SerializeImagePulls: utils.BoolPtr(false),
}
```